docs/references/javascript/organization-domain.mdx

```diff
@@ -76,9 +70,3 @@ function attemptAffiliationVerification(params: AttemptAffiliationVerificationPa
 | Name | Type | Description |
 | --- | --- | --- |
 | `code` | `string` | The one-time code that was sent to the user as part of this verification step. |
-
-### `attemptAffiliationVerification()` returns

```

Comment:
Why do we delete the "returns" headings and content here but keep them in other files?